rllib/connectors/action.py

```diff
@@ -0,0 +1,173 @@
+from typing import Any, List
+
+from ray.rllib.connectors.connector import (
+    Connector,
+    ConnectorContext,
+    ConnectorPipeline,
+    ActionConnector,
+    register_connector,
+    get_connector,
+)
+from ray.rllib.utils.annotations import DeveloperAPI
+from ray.rllib.utils.numpy import convert_to_numpy
+from ray.rllib.utils.spaces.space_utils import (
+    clip_action,
+    get_base_struct_from_space,
+    unsquash_action,
+    unbatch,
+)
+from ray.rllib.utils.typing import (
+    ActionConnectorDataType,
+    TrainerConfigDict,
+)
+
+
+@DeveloperAPI
+class ConvertToNumpyConnector(ActionConnector):
+    def __call__(self, ac_data: ActionConnectorDataType) -> ActionConnectorDataType:
```

Comment:
This and the next one miss the __ init __ method. Are we sure `from_context` works without them?

Reply:
yeah, they will just use the base class's __init__. I have unit tests for it :)
but even better, these stateless single function connectors should be implemented using LambdaConnectors now.